src/biokbase/narrative/jobs/appmanager.py

```diff
@@ -1093,5 +1093,6 @@ def _get_agent_token(self, name: str) -> Dict[str, str]:
     def register_new_job(self, job: Job) -> None:
         JobManager().register_new_job(job)
         self._send_comm_message("new_job", {"job_id": job.job_id})
-        JobComm().lookup_job_state(job.job_id)
-        JobComm().start_job_status_loop()
+        with exc_to_msg("appmanager"):
```

Comment:
So this was 1 of 3 "entry points" into the JobComm code. Just put the `exc_to_msg` context manager here.
The 1 "entry point" that is not covered with `exc_to_msg` is when `jc.start_job_status_loop` is called from the FE